### PR TITLE
[i18n-ar] Add file `docs/source/ar/model_doc/xmod.md` 

### PR DESCRIPTION
**Pull request by AhmedAlmaghz**
# What does this PR do? 
 Translate docs into Arabic issue-number #32435 
 @stevhliu , @MKhalusova , @abodacs 
 This pull request contains changes [**for file : docs/source/ar/model_doc/xmod.md**](https://github.com/AhmedAlmaghz/transformers/blob/Add_docs_source_ar_model_doc_xmod_md/docs/source/ar/model_doc/xmod.md)

### CODE REVIEW

File: docs/source/ar/model_doc/xmod.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
+# X-MOD
+
+## نظرة عامة
+اقترح نموذج X-MOD في ورقة "رفع لعنة تعدد اللغات عن طريق التدريب المسبق للمحولات المعيارية" من قبل Jonas Pfeiffer وآخرون. يوسع X-MOD نماذج اللغة المقنعة متعددة اللغات مثل XLM-R لتشمل مكونات معيارية خاصة باللغة (مهايئات اللغة) أثناء التدريب المسبق. بالنسبة للتعديل الدقيق، يتم تجميد مهايئات اللغة في كل طبقة من المحول.
```
Comment (stevhliu) [inline docs/source/ar/model_doc/xmod.md:4]: I think you're missing a link to the paper and model here

File: docs/source/ar/model_doc/xmod.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+ملخص الورقة هو كما يلي:
+
+> "من المعروف أن النماذج متعددة اللغات المُدربة مسبقًا تعاني من لعنة تعدد اللغات، والتي تسبب انخفاض الأداء لكل لغة كلما غطت المزيد من اللغات. نعالج هذه المشكلة من خلال تقديم وحدات خاصة باللغة، مما يسمح لنا بزيادة السعة الإجمالية للنموذج، مع الحفاظ على إجمالي عدد المعلمات القابلة للتدريب لكل لغة ثابتة. على عكس العمل السابق الذي يتعلم المكونات الخاصة باللغة بعد التدريب، نقوم بتدريب وحدات نماذجنا المعيارية متعددة اللغات من البداية. تُظهر تجاربنا على الاستدلال اللغوي الطبيعي، وتعريف الكيانات المسماة، والإجابة على الأسئلة أن نهجنا لا يخفف من التداخل السلبي بين اللغات فحسب، بل يمكّن أيضًا من النقل الإيجابي، مما يؤدي إلى تحسين الأداء أحادي اللغة وعبر اللغات. علاوة على ذلك، يمكّن نهجنا من إضافة لغات بعد التدريب دون انخفاض ملحوظ في الأداء، مما لم يعد يقتصر استخدام النموذج على مجموعة اللغات المدربة مسبقًا."
+
+تمت المساهمة بهذا النموذج من قبل jvamvas. يمكن العثور على الكود الأصلي والتوثيق الأصلي في الروابط المذكورة أعلاه.
```
Comment (stevhliu) [inline docs/source/ar/model_doc/xmod.md:10]: Missing a link to the contributors profile and the original codebase

**Review by stevhliu — COMMENTED** [review]
Thanks, you'll need to add this to the `toctree`

**Comment by AhmedAlmaghz** [comment]
https://github.com/huggingface/transformers/pull/32644#issuecomment-2289859120

**Pull request closed without merging**